app/src/main/java/com/facebook/encapp/BufferTranscoder.java

```diff
@@ -236,34 +236,34 @@ public String start() {
             mStats.setCodec(mCodec.getName());
         }
 
-    try {
-        MediaFormat mediaFormat = mDecoder.getOutputFormat();
-        Log.d(TAG, "Decoder output format");
-        logMediaFormat(mediaFormat);
-        mediaFormat = TestDefinitionHelper.mergeEncoderSettings(mTest, mediaFormat);
-        //TODO: color handling
-        // The decoder must output same colorformat as input. QC hw encoder does not take same pix fmt android sw decoder.
-        mediaFormat.setInteger(MediaFormat.KEY_COLOR_FORMAT, matchingColor);
-        mStats.pushTimestamp("encoder.configure");
-        mCodec.configure(
-                mediaFormat,
-                null /* surface */,
-                null /* crypto */,
-                MediaCodec.CONFIGURE_FLAG_ENCODE);
-        mStats.pushTimestamp("encoder.configure");
-        Log.d(TAG, "Check input format after encoder is configured");
-        logMediaFormat(mCodec.getInputFormat());
-
-        mMuxer = createMuxer(mCodec, mediaFormat);
         try {
-            Log.d(TAG, "Start encoder");
-            mStats.pushTimestamp("encoder.start");
-            mCodec.start();
-            mStats.pushTimestamp("encoder.start");
-        } catch (Exception ex) {
-            Log.e(TAG, "Start failed: " + ex.getMessage());
-            //return "Start encoding failed";
-        }
+            MediaFormat mediaFormat = mDecoder.getOutputFormat();
+            Log.d(TAG, "Decoder output format");
+            logMediaFormat(mediaFormat);
+            mediaFormat = TestDefinitionHelper.mergeEncoderSettings(mTest, mediaFormat);
+            //TODO: color handling
+            // The decoder must output same colorformat as input. QC hw encoder does not take same pix fmt android sw decoder.
+            mediaFormat.setInteger(MediaFormat.KEY_COLOR_FORMAT, matchingColor);
+            mStats.pushTimestamp("encoder.configure");
+            mCodec.configure(
+                    mediaFormat,
+                    null /* surface */,
+                    null /* crypto */,
```

Comment:
This looks like it belongs to a different commit since it seems to have little to do with lcevc. Break it out and make a pull request for generic updates, please.